ddr/tools/blob_reader/blob_reader.cpp

```diff
@@ -294,7 +294,7 @@ main(int argc, char *argv[])
 			currentStruct += sizeof(BlobStruct);
 
 			Structure *builtStruct = (Structure *)malloc(sizeof(Structure));
-			memset(builtStruct, 0, sizeof(Structure));
+			memset((void *)builtStruct, 0, sizeof(Structure));
```

Comment:
I can not see why this code would generate that warning.  Structure is a simple Struct.... I thought this warning was for non-POD types

Reply:
I think the fields of type `vector` declassify it as a POD type.